Mathlib/MeasureTheory/Measure/WithDensityFinite.lean

```diff
@@ -45,33 +47,47 @@ namespace MeasureTheory
 
 variable {α : Type*} {mα : MeasurableSpace α}
 
-/-- A finite measure obtained from an s-finite measure `μ`, such that
-`μ = μ.toFinite.withDensity μ.densityToFinite` (see `withDensity_densitytoFinite`). -/
-noncomputable def Measure.toFinite (μ : Measure α) [SFinite μ] : Measure α :=
+/-- Auxiliary definition for `MeasureTheory.Measure.toFinite`. -/
+noncomputable def Measure.toFiniteAux (μ : Measure α) [SFinite μ] : Measure α :=
   Measure.sum (fun n ↦ (2 ^ (n + 1) * sFiniteSeq μ n Set.univ)⁻¹ • sFiniteSeq μ n)
 
-lemma toFinite_apply (μ : Measure α) [SFinite μ] (s : Set α) :
-    μ.toFinite s = ∑' n, (2 ^ (n + 1) * sFiniteSeq μ n Set.univ)⁻¹ * sFiniteSeq μ n s := by
-  rw [Measure.toFinite, Measure.sum_apply_of_countable]; rfl
+/-- A finite measure obtained from an s-finite measure `μ`, such that
+`μ = μ.toFinite.withDensity μ.densityToFinite` (see `withDensity_densitytoFinite`).
+If `μ` is non-zero, this is a probability measure. -/
+noncomputable def Measure.toFinite (μ : Measure α) [SFinite μ] : Measure α :=
+  (μ.toFiniteAux Set.univ)⁻¹ • μ.toFiniteAux
```

Comment:
It might be good to define this using [ProbabilityTheory.cond](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Probability/ConditionalProbability.html#ProbabilityTheory.cond) which gives finite measure for free